addon/components/materialize-switch.js

```diff
@@ -0,0 +1,21 @@
+import Ember from 'ember';
+import layout from '../templates/components/materialize-switch';
+
+export default Ember.Component.extend({
+  layout: layout,
+  classNames: ['switch'],
+  labelOn: 'On',
+  labelOff: 'Off',
+  valueOn: true,
+  valueOff: false,
+  value: false,
+  isChecked: function (key, val) {
```

Comment:
switches/checkboxes/radios are still in flux a little bit. I want to use the same component as the itemViewClass for a set of switches/checks/radios, and for single switches/checks/radios. It's a bit of a long rabbit hole to go down, but should result some really nice abstractions that developers can build on (i.e., multi-select from a list of image thumbnails)